docs/rum_mobile_vitals.md

```diff
@@ -0,0 +1,41 @@
+## Overview
+
+Real User Monitoring offers Mobile Vitals, a set of metrics based on [Android Vitals][1] that can help you compute insights about your mobile application's responsiveness, stability, and resource consumption. 
+
+![Mobile Vitals in the RUM Explorer][2]
+
+Mobile vitals appear in your application's **Overview** tab and in the side panel under **Performance** > **Event Timings and Mobile Vitals** when you click on an individual view in the [RUM Explorer][3]. Click on a graph in **Mobile Vitals** to apply a filter by version or examine filtered sessions. 
+
+![Event Timings and Mobile Vitals][4]
+
+Mobile vitals include recommended benchmark ranges that correlate directly to your application's user experience. You can see where a metric scores on the range and click **Search Views With Poor Performance** to apply a filter in your search query.
+
+## Metrics
+
+The following metrics provide insight into your mobile application's performance.
+
+| Measurement                    | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+|--------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Slow renders                   | To ensure a smooth, [jank-free][5] user experience, your application should render frames in under 60 seconds. <br /><br />  RUM tracks the application’s [display refresh rate][6] using `@view.refresh_rate_average` and `@view.refresh_rate_min` view attributes. <br /><br />  With slow rendering, you can monitor which views are taking longer than 40, 50, or 60 seconds to render. |
```

Comment:
Frame should be rendered under 16 ms (or with frequency more than 60 Hz). What we show in mobile vitals is frame rate, kind of in Hz, but these are normalized Hz, so even if device frame rate is 120 Hz, max cap shown will still be 60 Hz. Should we mention normalization thing? cc @xgouchet 

Considering that, the following sentence should be rewritten:

```
With slow rendering, you can monitor which views are taking longer than 40, 50, or 60 seconds to render.
```

We should say something like `which views are taking longer than 16 ms to render`. 40 Hz will be 25 ms, 50 Hz will be 20 ms. But honestly the only value I would mention here is 16ms/60 Hz.

Reply:
Good catch indeed. 
And it might be worth it adding a note along the lines of:
> **Note**: the actual refresh rates are normalized on a 0-60 fps range (e.g.: if your app runs at 100fps on a device capable of rendering 120fps, it will be reported as 50fps in the mobile vitals).

Reply:
```suggestion
| Slow renders                   | To ensure a smooth, [jank-free][5] user experience, your application should render frames in under 60 Hz. <br /><br />  RUM tracks the application’s [display refresh rate][6] using `@view.refresh_rate_average` and `@view.refresh_rate_min` view attributes. <br /><br />  With slow rendering, you can monitor which views are taking longer than 16 ms or 60 Hz to render. **Note:** Refresh rates are normalized on a range of zero to 60 fps. For example, if your application runs at 100 fps on a device capable of rendering 120 fps, Datadog reports 50 fps in the Mobile Vitals. |
```